gtk/gtk-play.c

```diff
@@ -207,6 +207,58 @@ gtk_play_set_rate (GtkPlay * play, gdouble step)
   }
 }
 
+gboolean
+key_press_event_cb (GtkWidget * widget, GdkEventKey * event, gpointer data)
+{
+  GtkPlay *play = (GtkPlay *) widget;
+
+  if (event->state != 0 || event->type != GDK_KEY_PRESS)
+    return FALSE;
+
+  switch (event->keyval) {
+    case GDK_KEY_F11:
+    case GDK_KEY_f:
+    case GDK_KEY_F:
+      gtk_toggle_button_set_active (GTK_TOGGLE_BUTTON (play->fullscreen_button),
+          TRUE);
+      break;
+    case GDK_KEY_space:
+      gtk_button_clicked (GTK_BUTTON (play->play_pause_button));
+      break;
+    case GDK_KEY_q:
+    case GDK_KEY_Q:
+    case GDK_KEY_Escape:
+      gtk_toggle_button_set_active (GTK_TOGGLE_BUTTON (play->fullscreen_button),
+          FALSE);
+    case GDK_KEY_Right:
+      if (g_list_next (play->current_uri))
+        gtk_button_clicked (GTK_BUTTON (play->next_button));
+      break;
+    case GDK_KEY_Left:
+      if (g_list_previous (play->current_uri))
+        gtk_button_clicked (GTK_BUTTON (play->prev_button));
```

Comment:
Maybe it would be better to use the same keybindings as other media players (totem and snappy and mplayer for example) use, e.g. cursor keys for seeking.